doc/gitlab.nvim.txt

```diff
@@ -187,6 +187,7 @@ you call this function with no values the defaults will be used:
         temp_registers = {}, -- List of registers for backing up popup content (see `:h gitlab.nvim.temp-registers`)
       },
       discussion_tree = { -- The discussion tree that holds all comments
+        chevrons = { " ", " ", "  " },
```

Comment:
Hi @c0r73x, you could add some description to the newly added setting. It's not obvious what the third string is for (it's indentation, right?). Also, maybe "chevrons" may not be the most explicit name - imagine somebody decides to use something like `+` and `-` instead. E.g., [neo-tree](https://github.com/nvim-neo-tree/neo-tree.nvim) uses `expander_collapsed` and `expander_expanded`. You might consider changing this to something like (and add some more documentation):
```suggestion
        expanders = {
          expanded = " ", 
          collapsed = " ",
          indentation = "  ",
        },
```

Reply:
Done 👍